index.html

```diff
@@ -3005,8 +3016,14 @@ <h5>Uses beyond mapping</h5>
 
 <h5>Related web specifications</h5>
 <p>
-<!-- Any existing/proposed specs that address similar needs or otherwise interact.
-     Include links with proper ReSpec references. -->
+<a data-cite="SVG2#">SVG</a> contains both a <code>polyline</code> and a <code>polygon</code> element.
+A comma separated list of coordinates is assigned to the <code>points</code> HTML attribute to generate the shape.
```

Comment:
`points` is only defined in SVG so I don't think this should say "the `points` HTML attribute" but either "SVG attribute" or just "attribute" since we're already talking about SVG here?